include/deal.II/grid/filtered_iterator.h

```diff
@@ -525,10 +525,15 @@ namespace IteratorFilters
  * @ingroup Iterators
  * @author Wolfgang Bangerth, 2002
  */
-template <typename BaseIterator>
-class FilteredIterator : public BaseIterator
+template <typename BaseIterator_>
+class FilteredIterator : public BaseIterator_
```

Comment:
Just call it `BaseIteratorType` instead of using the awkward underscore. Though I think you don't actually need the change any more because you could rid of the new `using` declaration a couple of lines below, right?

Reply:
Ah, yes... I originally used this change on [this line](https://github.com/dealii/dealii/pull/7911/files#diff-e4bd193cb40dfd273fb5ee9d747013e6R116) but its indeed no longer needed.